packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
why moving no explicit any to top of file.

Reply:
because there were multiple line wise any. Files with a single line any disabled, did not make any change like MultiStackedBarChart.

Reply:
what is the difference between the 2 representations. Which is better?